doc/release_notes_1.0.rst

```diff
@@ -0,0 +1,478 @@
+   After booting UOS with multiple USB devices plugged in, there's a 60% chance that one or more devices are not discovered.
+
+   **Impact:** Cannot use multiple USB devices at same time.
+
+   **Workaround:** Unplug and plug-in the unrecognized device after booting.
+
+-----
+
+:acrn-issue:`1991` - Input not accepted in UART Console for corner case
+   Input is useless in UART Console for a corner case, demonstrated with these steps:
+
+   1) Boot to SOS
+   2) ssh into the SOS.
+   3) use ``./launch_UOS.sh`` to boot UOS.
+   4) On the host, use ``minicom -s dev/ttyUSB0``.
+   5) Use ``sos_console 0`` to launch SOS.
+
+   **Impact:** Fails to use UART for input.
+
+   **Workaround:** Enter other keys before typing :kbd:`Enter`.
+
+-----
+
+:acrn-issue:`2267` - [APLUP2][LaaG] LaaG can't detect 4k monitor
+   After launching UOS on APL UP2 , 4k monitor cannot be detected.
+
+   **Impact:** UOS can't display on a 4k monitor.
+
+   **Workaround:** Use a monitor with less than 4k resolution.
+
+-----
+
+:acrn-issue:`2278` - [KBLNUC] Cx/Px is not supported on KBLNUC
+   C states and P states are not supported on KBL NUC.
+
+   **Impact:** Power Management state-related operations in SOS/UOS on
+   KBL NUC can't be used.
+
+   **Workaround:** None
+
+-----
+
+:acrn-issue:`2279` - [APLNUC] After exiting UOS, SOS can't use USB keyboard and mouse
+   After exiting UOS with mediator
+   Usb_KeyBoard and Mouse, SOS cannot use the USB keyboard and mouse.
+
+   These steps reproduce the issue:
+
+   1) Insert USB keyboard and mouse in standard A port (USB3.0 port)
+   2) Boot UOS by sharing the USB keyboard and mouse in cmd line:
+
+      ``-s n,xhci,1-1:1-2:1-3:1-4:2-1:2-2:2-3:2-4 \``
+
+   3) UOS access USB keyboard and mouse.
+   4) Exit UOS.
+   5) SOS tries to access USB keyboard and mouse, and fails.
+
+   **Impact:** SOS cannot use USB keyboard and mouse in such case.
+
+   **Workaround:** Unplug and plug-in the USB keyboard and mouse after exiting UOS.
+
+-----
+
+:acrn-issue:`2527` - System will crash after a few minutes running stress test ``crashme`` tool in SOS/UOS.
+   System stress test may cause a system crash.
+
+   **Impact:** System may crash in some stress situations.
+
+   **Workaround:** None
+
+-----
+
+:acrn-issue:`2526` - Hypervisor crash when booting UOS with acrnlog running with mem loglevel=6
+   If we use ``loglevel 3 6`` to change the mem loglevel to 6, we may hit a page fault in HV.
+
+   **Impact:** Hypervisor may crash in some situation.
+
+   **Workaround:** None
+
+-----
+
+:acrn-issue:`2753` - UOS cannot resume after suspend by pressing power key
+   UOS cannot resume after suspend by pressing power key
+
+   **Impact:** UOS may failed to resume after suspend by pressing the power key.
+
+   **Workaround:** None
+
+
+
+Change Log
+**********
+
+These commits have been added to the acrn-hypervisor repo since the v0.6
```

Comment:
Does this ``since the v0.6 release`` need to be changed?  You certainly don't want to list all commits since the 0.1 release though.